### PR TITLE
Add documentation for security-sensitivity-level

### PR DESCRIPTION
**Pull request by Gabeblis**
## Description 

Added documentation regarding the security sensitivity level. This documentation includes a description of requirements/expectations, an XML example of the requirements/expectations, and some important notes.

### CODE REVIEW

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md [hunk 1]
```diff
@@ -300,7 +300,7 @@ Valid IAL, AAL, and FAL values (as defined by NIST SP 800-63):
 ---
 ### System Sensitivity Level 
 
-The privacy system designation in in the FedRAMP SSP template document, illustrated in the figure below, is expressed through the following core OSCAL property.
+The privacy system designation in the FedRAMP SSP template, illustrated in the figure below, is expressed through the following core OSCAL property. Specifically, a FedRAMP SSP must define its FIPS-199 system sensitivity level as one of the allowed values below, and it should match the highest security impact level for the system’s confidentiality, integrity, and availability objectives.
```
Comment (aj-stein-gsa) [inline content/documentation/ssp/4-ssp-template-to-oscal-mapping.md:303]: I believe @Rene2mt called this out yesterday in the paired constraint PR review, but I thought it was a should, not a must.

```suggestion
The privacy system designation in the FedRAMP SSP template, illustrated in the figure below, is expressed through the following core OSCAL property. Specifically, a FedRAMP SSP should define its FIPS-199 system sensitivity level as one of the allowed values below, and it should match the highest security impact level for the system’s confidentiality, integrity, and availability objectives.
```

**Review by aj-stein-gsa — CHANGES_REQUESTED** [review]

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md (hunk 1 above)
Comment (kyhu65867) [inline content/documentation/ssp/4-ssp-template-to-oscal-mapping.md:303]: Not your fault, just a nature of the way this was initially structured, but I think this section could be streamlined to be more concise and clear. "The privacy system designation in the FedRAMP SSP template, shown in the figure below, can be expressed through a core OSCAL property that specifies the FIPS-199 sensitivity level based on the highest security impact for confidentiality, integrity, and availability."

**Review by kyhu65867 — CHANGES_REQUESTED** [review]
Slight tweak

**Review by DimitriZhurkin — COMMENTED** [review]
Absolutely not a blocker, but I think `the figure below` should be `the table below`. Otherwise, looks good to me.

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md (hunk 1 above)
Comment (Gabeblis) [reply content/documentation/ssp/4-ssp-template-to-oscal-mapping.md]: Noted and change applied. Thank you.

Comment (Gabeblis) [reply content/documentation/ssp/4-ssp-template-to-oscal-mapping.md]: Thank you for the feedback. Addressed: 978cf11585c6b1ec8c6ad3ae74547a9310a65ce8

**Review by aj-stein-gsa — COMMENTED** [review]
Have I been so slow on another PR that we are writing against the same gap in the system characteristics?

https://github.com/GSA/automate.fedramp.gov/pull/56#pullrequestreview-2382601190

**Comment by Gabeblis** [comment]
> Have I been so slow on another PR that we are writing against the same gap in the system characteristics?
> 
> [#56 (review)](https://github.com/GSA/automate.fedramp.gov/pull/56#pullrequestreview-2382601190)

Fun. There is definitely a little overlap here, but there is new information being added. I can definitely work on merging this into the existing PR, and getting that to a good state.

**Comment by aj-stein-gsa** [comment]
> > Have I been so slow on another PR that we are writing against the same gap in the system characteristics?
> 
> > 
> 
> > [#56 (review)](https://github.com/GSA/automate.fedramp.gov/pull/56#pullrequestreview-2382601190)
> 
> 
> 
> Fun. There is definitely a little overlap here, but there is new information being added. I can definitely work on merging this into the existing PR, and getting that to a good state.

I'm fine either way. I've been too slow and it's catching up with me and the scope of that issue was vague. So ... I broke my own principles there. Feel free to repoint to my branch if you want.

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md [hunk 2]
```diff
@@ -300,7 +300,7 @@ Valid IAL, AAL, and FAL values (as defined by NIST SP 800-63):
 ---
 ### System Sensitivity Level 
 
-The privacy system designation in in the FedRAMP SSP template document, illustrated in the figure below, is expressed through the following core OSCAL property.
+The privacy system designation in the FedRAMP SSP template, illustrated in the figure below, can be expressed through a core OSCAL property that defines its FIPS-199 system sensitivity level as one of the allowed values below. The FIPS-199 system sensitivity level should match the highest security impact level for the system’s confidentiality, integrity, and availability objectives.
```
Comment (kyhu65867) [inline content/documentation/ssp/4-ssp-template-to-oscal-mapping.md:303]: I like this wording a lot better

**Review by kyhu65867 — APPROVED** [review]
Looks good to me! Thanks for the hard work :)

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md (hunk 2 above)
Comment (Rene2mt) [inline content/documentation/ssp/4-ssp-template-to-oscal-mapping.md:303]: Just noticed the site has a minor error here, so we can fix it while we're updating this section.

```suggestion
The FIPS PUB 199 level designation in the FedRAMP SSP template, illustrated in the figure below, can be expressed through a core OSCAL property that defines its FIPS-199 system sensitivity level as one of the allowed values below. The FIPS-199 system sensitivity level should match the highest security impact level for the system’s confidentiality, integrity, and availability objectives.
```

**Review by Rene2mt — CHANGES_REQUESTED** [review]
PR changes look good.  Added a suggestion to fix a small pre-existing issue while we're updating this section

**Review by DimitriZhurkin — APPROVED** [review]

**Review by Rene2mt — APPROVED** [review]

**Comment by aj-stein-gsa** [comment]
@Gabeblis I am really sorry I did not review it on Friday. Thanks for pinging me again. I actually did not complete a review then, but no time like the present. I gave some feedback.

**Comment by Gabeblis** [comment]
> @Gabeblis I am really sorry I did not review it on Friday. Thanks for pinging me again. I actually did not complete a review then, but no time like the present. I gave some feedback.

Thank you for the feedback sir

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md [hunk 3]
```diff
@@ -300,7 +300,7 @@ Valid IAL, AAL, and FAL values (as defined by NIST SP 800-63):
 ---
 ### System Sensitivity Level 
 
-The privacy system designation in in the FedRAMP SSP template document, illustrated in the figure below, is expressed through the following core OSCAL property.
+The FIPS PUB 199 level designation in the FedRAMP SSP template, illustrated in the figure below, can be expressed through a core OSCAL property that defines its FIPS-199 system sensitivity level as one of the allowed values below. The FIPS-199 system sensitivity level should match the highest security impact level for the system’s confidentiality, integrity, and availability objectives.
```
[the remark this answers is not in the document]
Comment (DimitriZhurkin) [reply content/documentation/ssp/4-ssp-template-to-oscal-mapping.md]: Sorry, but I personally find it rather difficult to read and understand.

[the remark this answers is not in the document]
Comment (aj-stein-gsa) [reply content/documentation/ssp/4-ssp-template-to-oscal-mapping.md]: Do you have a specific suggestion on how you would edit it then?

[the remark this answers is not in the document]
Comment (DimitriZhurkin) [reply content/documentation/ssp/4-ssp-template-to-oscal-mapping.md]: Just a quick hack:

A FedRAMP SSP defines the system's sensitivity level in `security-sensitivity-level`. The supporting information for the sensitivity level is specified in `security-impact-level`.

The value of `security-sensitivity-level` should match the highest security level specified in `security-impact-level` (`security-objective-confidentiality`, `security-objective-integrity`, and `security-objective-availability`).

For example, if the highest `security-impact-level` value is `fips-199-moderate`, the `security-sensitivity-level` value should be `fips-199-moderate`.

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md [hunk 4]
```diff
@@ -300,7 +300,7 @@ Valid IAL, AAL, and FAL values (as defined by NIST SP 800-63):
 ---
 ### System Sensitivity Level 
 
-The privacy system designation in in the FedRAMP SSP template document, illustrated in the figure below, is expressed through the following core OSCAL property.
+A FedRAMP SSP in OSCAL defines the system's sensitivity level and supporting information with `security-sensitivity-level` and `security-impact-level` adjacent to one another, not in a separate attachment like the SSP Appendix K FIPS Pub Level document, as illustrated below. The security sensitivity level should match the highest security impact level for the system’s confidentiality, integrity, and availability objectives for its `security-impact-level`.
```
Comment (kyhu65867) [inline content/documentation/ssp/4-ssp-template-to-oscal-mapping.md:303]: Adjacent to me sounds like side-by-side. I would say consecutively but I think that word may not be the most accessible so for Plain Language sake I would say "A FedRAMP SSP in OSCAL defines the system's sensitivity level and supporting information with 'security-sensitivity-level' directly followed by 'security-impact-level', NOT in a separate attachment....."

**Review by kyhu65867 — CHANGES_REQUESTED** [review]

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md (hunk 4 above)
Comment (Gabeblis) [reply content/documentation/ssp/4-ssp-template-to-oscal-mapping.md]: I see how the wording is a little misleading here. Resolved: 948dcd1a78b5afb18b03765cd3b11f25bfd4bd39

**Review by aj-stein-gsa — APPROVED** [review]

**Review by kyhu65867 — APPROVED** [review]
This looks better and more clear, thanks!

**Pull request merged**